getting-started.md

```diff
@@ -0,0 +1,64 @@
+# **Getting Started**
+
+> Ok, I have installed Allstar on my account/organization, now what?
+
+By default, Allstar will not take any actions to avoid unwanted noise. To
+quickly enable Allstar on all of your repos:
+
+1. Create a repository named `.allstar`
+1. Create a file named `allstar.yaml` with the contents
+
+       optConfig:
+         optOutStrategy: true
+
+1. Create a file named `branch_protection.yaml` with the contents
+
+       optConfig:
+         optOutStrategy: true
+       action: issue
+
+This will enable Allstar and the Branch Protection policy on all repos with the
+default settings. The `issue` action will create GitHub issues in each repo for
+violations of the security policy.
+
+For all the available options in `allstar.yaml` see the comments on [the config
+definition
+here](https://github.com/ossf/allstar/blob/main/pkg/config/config.go#L29-L50). Similarly
```

Comment:
ideally should move these settings into its own file, so it is cleaner and then dont need to reference things by line numbers.